uint32 timestamp
float64 velocity
float64 steering_angle